### PR TITLE
Add Account tab in Space setings

### PR DESCRIPTION
**Pull request by EmiliaPavlova**
Added Account tab on Root Space settings with:
- URL, Visibility and Host information blocks
- Delete block

### CODE REVIEW

File: src/domain/journey/space/pages/SpaceAccount/SpaceAccountView.tsx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        },
+      },
+    });
+  };
+
+  const ALKEMIO_DOMAIN = 'https://alkem.io/';
+  const loading = deletingSpace && spacePriviledgesLoading && hostOrganizationLoading;
+  const avatar = hostOrganization?.profile.avatar?.uri;
+  const location = hostOrganization?.profile.location;
+  console.log(t('pages.admin.generic.sections.account.contactsLink'));
```
Comment (bobbykolev) [inline src/domain/journey/space/pages/SpaceAccount/SpaceAccountView.tsx:77]: dev console log to be removed

File: src/domain/journey/space/pages/SpaceAccount/SpaceAccountView.tsx [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+              entity={t('common.space')}
+              open={openDeleteDialog}
+              onClose={closeDialog}
+              onDelete={() => handleDelete(journeyId)}
+              submitting={deletingSpace}
+            />
+          )}
+        </>
+      )}
+    </PageContent>
```
Comment (bobbykolev) [inline src/domain/journey/space/pages/SpaceAccount/SpaceAccountView.tsx:149]: let's add loading here:

` {loading && (
      <Box marginX="auto">
        <CircularProgress />
      </Box>
    )}`

File: src/core/i18n/en/translation.en.json [hunk 3]
```diff
@@ -1351,7 +1352,7 @@
       },
       "read-access": "Allow read access for non-space members to Subspaces",
       "about": "About this Space",
-      "visibility": "This Space is currently in <strong>{{visibility}}</strong> mode",
+      "visibility": "This Space is currently in <strong>{{mode}}</strong> mode",
```
Comment (bobbykolev) [inline src/core/i18n/en/translation.en.json:1355]: Do we need this chagne? It looks like it's used in SpaceProfileFixedInformation.tsx and it's not changed to mode there.

Comment (EmiliaPavlova) [reply src/core/i18n/en/translation.en.json]: I wish I knew earlier that there is already the same section.

**Review by bobbykolev — APPROVED** [review]

**Pull request merged**